terraform/install.sh

```diff
@@ -333,12 +333,12 @@ parseArguments() {
       shift
       ;;
     -h|--help)
-      log "Deploy Stackdriver Sandbox to a GCP project"
+      log "Deploy Cloud Ops Sandbox to a GCP project"
```

Comment:
Are we rebranding to Cloud Ops Sandbox or Cloud Operations Sandbox? I'm seeing conflicting names here and in the Github Repo Description

Reply:
The name of the project is "Cloud Operations Sandbox" and this is how we should refer to it in free form text (Readmes, docs). However in code (e.g. the name of the gke cluster) is supposed to be "cloud-ops-sandbox" instead of "stackdriver-sandbox".